src/simpleTrianglePatterns/simpleTrianglePattern2.java

```diff
@@ -4,9 +4,9 @@
 
 public class simpleTrianglePattern2 {
     public static void main(String[] args){
-        Scanner scan = new Scanner(System.in);
```

Comment:
Is scan a meaningful name?

Reply:
yes.

Reply:
Java fields must be nouns, like you cannot describe a state with a verb, and object fields are states.